### ISSUE TITLE
Uncaught TypeError: o.get is not a function

### ISSUE THREAD

**Issue by VivMendes376** [atom/atom#19861]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.40.1 x64
**Electron**: 3.1.10
**OS**: Microsoft Windows 10 Home
**Thrown From**: Atom Core

### Stack Trace

Uncaught TypeError: o.get is not a function

```
At C:\Users\Utilizador\AppData\Local\atom\app-1.40.1\resources\app\static\<embedded>:11

TypeError: o.get is not a function
    at decorationCountsByLayer.forEach (~/AppData/Local/atom/app-1.40.1/resources/app/static/<embedded>:11:518047)
    at Map.forEach (<anonymous>)
    at DecorationManager.decorationPropertiesByMarkerForScreenRowRange (~/AppData/Local/atom/app-1.40.1/resources/app/static/<embedded>:11:517809)
    at TextEditorComponent.queryDecorationsToRender (~/AppData/Local/atom/app-1.40.1/resources/app/static/<embedded>:11:82388)
    at TextEditorComponent.updateSyncBeforeMeasuringContent (~/AppData/Local/atom/app-1.40.1/resources/app/static/<embedded>:11:70481)
    at TextEditorComponent.updateSync (~/AppData/Local/atom/app-1.40.1/resources/app/static/<embedded>:11:67993)
    at visible.suppressUpdates.updatedSynchronously.updateScheduled.i.getScheduler.updateDocument (~/AppData/Local/atom/app-1.40.1/resources/app/static/<embedded>:11:67021)
    at ViewRegistry.performDocumentUpdate (~/AppData/Local/atom/app-1.40.1/resources/app/static/<embedded>:11:325676)
```

### Commands

```
     -3:30.9.0 core:confirm (textarea.xterm-helper-textarea)
```

### Non-Core Packages

```
atom-clock 0.1.17 
atom-numbers 0.7.0 
build-ember 0.0.1 
daverona-laravel-syntax 0.0.1 
daverona-mellow-syntax 0.0.1 
daverona-slime-syntax 0.0.1 
edit-numbers 0.4.1 
ember-cli-helper 0.18.0 
ember-tabs 2.4.3 
framer-syntax 1.1.0 
framer-ui 2.0.0 
gibber 0.1.3 
GobbieGobGoo 1.0.4 
handsome-light-syntax 0.3.4 
hey-pane 1.2.0 
Hydrogen 2.12.0 
ide-ember 0.5.0 
indent-detective 0.4.0 
ink 0.10.12 
javascript-snippets 1.2.1 
julia-client 0.9.4 
language-better-junos 0.2.0 
language-julia 0.19.1 
language-juniper 2.0.0 
language-juniper-junos 1.0.0 
language-junos 0.0.2 
language-sugar-cube 1.0.0 
latex-completions 0.3.6 
lime-syntax 1.0.1 
linter-biber 0.1.0 
midnight-owl-syntax 0.1.0 
moonlight 0.11.0 
pigments 0.40.2 
rocket-dark-syntax 1.0.0 
rubber-ducky 1.3.6 
scroll-through-time 0.3.1 
summer-night-syntax 2.0.1 
tab-numbers 0.6.1 
teletype 0.13.4 
tool-bar 1.1.12 
uber-juno 0.2.0 
yuno-commit 0.3.1 
zentabs 0.8.9 
```

**Comment by rsese** [comment]
Thanks for the report!  Can you confirm a few things for us?

* What were you doing at the time of the error?
* Is the error is reproducible?
* If reproducible, do you see the error in safe mode (`atom --safe`)?

**Comment by VivMendes376** [comment]
I have solved the problem by unstalling Juno and Julia and re-installing everything once again. It is a painfull job because we have to install all libraries once again.

I think I know what caused the problem: I installed the package javascrip-snippets. After that, the problem started to appear, and it did not bother how many times I started and restarted Atom.

The problem appeared in the following way: when I tried to open file+settings+packages ... there it was ... and simply Atom stopped working.
